sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -119,7 +119,7 @@ case class TestFunctionWithTypeCheckFailure(
 
 case class UnresolvedTestPlan() extends UnresolvedLeafNode
 
-class AnalysisErrorSuite extends AnalysisTest with DataTypeErrorsBase {
+class AnalysisErrorSuite extends AnalysisTest with DataTypeErrorsBase with SharedSparkSession {
```

Comment:
To-do: Remove `SharedSparkSession` trait, accidentally added while experimenting.